bundles/org.openhab.binding.ecotouch/src/main/resources/OH-INF/i18n/ecotouch_de.properties

```diff
@@ -61,6 +61,8 @@ channel-type.ecotouch.enable_pv.label = Abschaltung PV
 channel-type.ecotouch.enable_warmwater.label = Abschaltung Warmwasser
 channel-type.ecotouch.enable_warmwater.description = Handabschaltung Warmwasserbetrieb
 channel-type.ecotouch.hysteresis_heating.label = Hysterese Heizung
+channel-type.ecotouch.hysteresis_warmwater.label = Hysterese Warmwasser
```

Comment:
Please revert changes to this file. Translations are made by crowin.
For reference: https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations